install/hst-install-ubuntu.sh

```diff
@@ -258,7 +260,7 @@ done
 eval set -- "$args"
 
 # Parsing arguments
-while getopts "a:w:v:j:k:m:M:g:d:x:z:Z:c:t:i:b:r:o:q:l:y:s:u:e:p:W:D:fh" Option; do
+while getopts "a:w:v:j:k:m:M:g:d:x:z:Z:c:t:i:b:r:o:q:l:y:s:u:e:p:W:D:fOh" Option; do
```

Comment:
Should be 
D:O:fh 

So far I can remember we had issues if we don't do it like it

Reply:
I do not know it exactly. But AFAIK you need this : if you have parameters after it. In case of this "override" I check only, if the option is set. You see, that "f" and "h" do not have : too. In case of "6" for IPV6 i make it with :
But you are right, I do not know, if my assumption is correct.

About the idea of this option: Yes it is very dangerous. And you are right, it can produce more issues, as it helps at end. Therefore you can discuss and decide, if we really need it. As I said above "for some cases it may be interesting".

About the install/uninstall scripts. I have really my own scripts and can post it as a separate PR here. But my uninstall method is very hard: I delete some directories, logs, uninstall deb packages partially with "purge". Since for my personal needs it is good. After it I have a nearby "clean" system. But I do not know, if it is really good for all users. It can also produce more problems at end, if someone screams here after such uninstall: "Where is my data?"